packages/web-react/src/components/Pagination/PaginationLinkDirectional.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ElementType, ForwardedRef, forwardRef } from 'react';
+import { SpiritPaginationLinkDirectionalProps } from '../../types';
+import { Button } from '../Button';
+import { Icon } from '../Icon';
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_PaginationLinkDirectional'] }] */
+const _PaginationLinkDirectional = <E extends ElementType = 'a'>(
+  props: SpiritPaginationLinkDirectionalProps<E>,
+  ref: ForwardedRef<HTMLButtonElement>,
+) => {
+  const { direction, label, ...restProps } = props;
+
+  const iconType = {
+    previous: 'chevron-left',
+    next: 'chevron-right',
+  };
+
+  return (
+    <Button color="secondary" isSquare {...restProps} ref={ref}>
```

Comment:
I thought that we were talking about the usage of `ButtonLink`, weren't we?

Reply:
done